src/category_theory/limits/shapes/binary_products.lean

```diff
@@ -362,14 +362,15 @@ section
 variables {C} [has_binary_products.{v} C]
 variables {D : Type u₂} [category.{v} D] [has_binary_products.{v} D]
 
-local attribute [tidy] tactic.case_bash
-
 -- FIXME deterministic timeout with `-T50000`
```

Comment:
Remove?

Reply:
No, this one is a still a problem with `-T50000`.